### PR TITLE
Cs fixes

### PR DESCRIPTION
**Pull request by Nyholm**
Im finally happy with the CS after this PR.

### CODE REVIEW

File: src/CodeGenerator/File/Printer.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        foreach ($class->getProperties() as $property) {
+            $type = $property->getType();
+            $def = (($property->getVisibility() ?: 'public') . ($property->isStatic() ? ' static' : '') . ' '
+                . ($type ? ($property->isNullable() ? '?' : '') . ($this->resolveTypes && $namespace ? $namespace->unresolveName($type) : $type) . ' ' : '')
+                . '$' . $property->getName());
+
+            $properties[] = Helpers::formatDocComment((string) $property->getComment())
+                . $def
+                . (null === $property->getValue() && !$property->isInitialized() ? '' : ' = ' . $this->dump($property->getValue(), \strlen($def) + 3)) // 3 = ' = '
+                . ';';
```
Comment (Nyholm) [inline src/CodeGenerator/File/Printer.php]: I changed this line from `. ";\n"`

File: src/CodeGenerator/File/Printer.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        }
+
+        $methods = [];
+        $methodDefinitions = $class->getMethods();
+        ksort($methodDefinitions);
```
Comment (Nyholm) [inline src/CodeGenerator/File/Printer.php]: I sorted methods.

Comment (jderusse) [reply src/CodeGenerator/File/Printer.php]: clever

**Comment by Nyholm** [comment]
This PR has 2 commits, one with the changes of the CS rules. The other commit is only automated fixes.

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**